.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+# Sync miner docs from https://github.com/tellor-io/telliot/tree/master/docs in miner-documentation folder in the current repo
+name: Sync miner docs
+
+on:
+  schedule: 
+  - cron: "0 0 * * *" # every 15 minutes. set to whatever interval you like
+
+
+jobs:
+  miner-docs-synce:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.x' # Version range or exact version of a Python version to use, using SemVer's version range syntax
+          architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
+
+      # Runs a set of commands using the runners shell
+      - name:  Sync
+        run: |
+          git clone https://github.com/tellor-io/telliot -b master
+          export TELLIOT_MD5=$(find ./telliot/docs  -type f -exec md5sum {} \; | md5sum | cut -d" " -f1)
+          export CURRENT_MD5=$(find $GITHUB_WORKSPACE/miner-documentation -type f -exec md5sum {} \; | md5sum | cut -d" " -f1)
+          if [ "$TELLIOT_MD5" != "$CURRENT_MD5" ];
+          then 
+            sed -i 's/(/(miner-documentation\//g' ./telliot/docs/INDEX.md
+            python .github/workflow/update-summary.py ./telliot/docs/INDEX.md
```

Comment:
Why is this needed?

Reply:
I put an INDEX.md file in Telliot docs folder which contains Telliot table of content. This way the main docs can be in sync with Telliot table of content and we don't need to manually edit the SUMMARY.md file every time the auto-sync process creates a PR.

Reply:
Aah yeah I see what you mean. Lets call it SUMMARY.md to be consistent and also when opening the Telliot dedicated gitbook space it will be used as well.

Reply:
Also in the Telliot repo we would need to add a `.gitbook.yaml`  to specify the root folder and the location of the summary there.

Reply:
Ok, then I changed the action to handle a SUMMARY.md file.